setup.cfg

```diff
@@ -153,16 +153,15 @@ colab =
 dev = 
     # Type check
     mypy
-    types-Markdown 
-    types-requests 
-    types-PyYAML 
+    typing_extensions; python_version < '3.8'
     # Test
     pytest
     responses
     tox
     coverage
     python-multipart
     psutil
+    pytest-random-order
```

Comment:
do we really want to use that given our sometimes tight memory situation?

Reply:
I don't know actually... how test randomization can increase memory consumption? Tests are randomized within their own suites anyway, so regardless of which test runs first, the shared fixtures will still be shared.